packages/@aws-cdk/aws-dynamodb/test/dynamodb.test.ts

```diff
@@ -486,154 +487,10 @@ test('fails if both replication regions used with customer managed CMK', () => {
   })).toThrow('TableEncryption.CUSTOMER_MANAGED is not supported by DynamoDB Global Tables (where replicationRegions was set)');
 });
 
-test('if an encryption key is included, decrypt permissions are also added for grantStream', () => {
```

Comment:
Not sure what this test is trying to verify. Dropping it entirely.

Reply:
But the assertion does not actually verify decrypt permission on the user and the associated policy, does it?

Reply:
Further analysis -

[`grantStreamRead()`](https://github.com/aws/aws-cdk/blob/a0bdffc423f25af37fd6ce3970167f947fd027a0/packages/%40aws-cdk/aws-dynamodb/lib/table.ts#L654-L656) calls `combinedGrant()` API without any `tableActions` configured.

This means that the key permissions [will not be updated](https://github.com/aws/aws-cdk/blob/a0bdffc423f25af37fd6ce3970167f947fd027a0/packages/%40aws-cdk/aws-dynamodb/lib/table.ts#L899-L903).

Seems right to drop this test. I'm guessing reading a stream does not require permissions to the key, only reading the table does.

Reply:
Huh, you're right. This test was never testing anything. Weird...